sdk/python/.gitignore

```diff
@@ -135,4 +135,5 @@ __pypackages__/
 # bin
 flet/bin/
 
-/playground
\ No newline at end of file
+/playground
+hello.py
```

Comment:
Could you please remove `hello.py`?

Reply:
There have been merge conflicts which I fixed but now there is some pdm problems. I am not sure what it is, but I'll try fixing it on Sunday.